src/app/content/components/SidebarControl.tsx

```diff
@@ -1,43 +1,104 @@
 import React from 'react';
 import { FormattedMessage } from 'react-intl';
 import { connect } from 'react-redux';
+import styled, { css } from 'styled-components';
+import { ListOl } from 'styled-icons/fa-solid/ListOl';
+import { contentFont, textRegularSize } from '../../components/Typography';
 import { AppState, Dispatch } from '../../types';
 import { assertString } from '../../utils';
 import * as actions from '../actions';
 import * as selectors from '../selectors';
+import { State } from '../types';
+import { toolbarIconColor } from './constants';
+import { toolbarIconStyles } from './Toolbar';
+import { styleWhenSidebarClosed } from './utils/sidebar';
 
-interface Props extends React.HTMLProps<HTMLButtonElement> {
-  isOpen: boolean;
+interface InnerProps {
+  isOpen: State['tocOpen'];
   onClick: () => void;
-  children: React.ReactNode;
+  className?: string;
 }
+interface MiddleProps {
+  isOpen: State['tocOpen'];
+  openToc: () => void;
+  closeToc: () => void;
+}
+
+// tslint:disable-next-line:variable-name
+const ListIcon = styled(ListOl)`
+  ${toolbarIconStyles};
+`;
+
+// tslint:disable-next-line:variable-name
+const ToCButtonText = styled.h3`
+  font-family: ${contentFont};
```

Comment:
do we have multiple fonts in REX (excluding the actual content since it's Roboto or whatever)

Reply:
not currently, but sometimes designers like to use different fonts for headings and stuff so i didn't want to limit our flexibility